examples/darwin-framework-tool/templates/tests/ciTests.json

```diff
@@ -56,6 +56,11 @@
         "Test_TC_TCTL_2_1",
         "Disabled because darwin-framework-tool does not support constraints arithmetic operations",
         "Test_TC_FLW_2_1",
-        "Test_TC_RH_2_1"
+        "Test_TC_RH_2_1",
+        "Test_TC_S_1_1",
```

Comment:
These are disabled because of the "arithmetic operations" bits?  Or for some other reason?